lib/kb_uploadmethods/Utils/BatchUtil.py

```diff
@@ -286,11 +286,16 @@ def batch_import_assemblies_from_staging(self, params):
         staging_subdir = params.get('staging_subdir')
         workspace_name = params.get('workspace_name')
         assembly_set_name = params.get('assembly_set_name')
+        print ("STAGING BEFORE=", staging_subdir)
 
-        if staging_subdir.startswith(self.user_id + '/' ):
+        if staging_subdir.startswith('/' + self.user_id + '/' ):
             temp = len(self.user_id) + 1
-            staging_subdir = staging_subdir[temp:]
-        
+            staging_subdir = staging_subdir.replace('/' + self.user_id + '/' , '/') 
```

Comment:
limit replace to 1 occurrence. In case `staging_subdir` has multiple user_id in it.